zerver/webhooks/bitbucket3/tests.py

```diff
@@ -95,16 +97,203 @@ def test_push_update_multiple_branches(self) -> None:
+        expected_message = """zura marked [PR #6]\
+(http://139.59.64.214:7990/projects/SBOX/repos/sandbox/pull-requests/6) as \"needs work\""""
+        self.send_and_test_stream_message("pull_request_needs_work",
+                                          expected_topic,
+                                          expected_message)
+
+    def test_pr_marked_as_needs_review_and_include_title(self) -> None:
+        expected_topic = "custom_topic"
+        expected_message = """zura marked [PR #6 sample_file: Add sample_file.txt.]\
+(http://139.59.64.214:7990/projects/SBOX/repos/sandbox/pull-requests/6) as \"needs work\""""
+        self.url = self.build_webhook_url(topic='custom_topic')
+        self.send_and_test_stream_message("pull_request_needs_work",
+                                          expected_topic,
+                                          expected_message)
+
+    def test_pull_request_reviewer_added(self) -> None:
+        expected_message = """hypro999 reassigned [PR #1](http://139.59.64.214:7990/projects\
+/SBOX/repos/sandbox/pull-requests/1) to [shimura](http://139.59.64.214:7990/users/shimura)"""
+        expected_topic = "sandbox / PR #1 Branch1"
+        self.send_and_test_stream_message("pull_request_add_reviewer",
+                                          expected_topic,
+                                          expected_message)
+
+    def test_pull_request_reviewer_added_and_include_title(self) -> None:
+        expected_message = """hypro999 reassigned [PR #1 Branch1](http://139.59.64.214:7990/projects\
+/SBOX/repos/sandbox/pull-requests/1) to [shimura](http://139.59.64.214:7990/users/shimura)"""
+        expected_topic = "custom_topic"
+        self.url = self.build_webhook_url(topic='custom_topic')
+        self.send_and_test_stream_message("pull_request_add_reviewer",
+                                          expected_topic,
+                                          expected_message)
+
+    def test_pull_request_reviewers_added(self) -> None:
+        expected_message = """hypro999 reassigned [PR #1](http://139.59.64.214:7990/projects\
+/SBOX/repos/sandbox/pull-requests/1) to [shimura](http://139.59.64.214:7990/users/shimura) and \
+[sougo](http://139.59.64.214:7990/users/sougo)"""
+        expected_topic = "sandbox / PR #1 Branch1"
+        self.send_and_test_stream_message("pull_request_add_two_reviewers",
+                                          expected_topic,
+                                          expected_message)
+
+    def test_pull_request_remove_all_reviewers(self) -> None:
+        expected_message = """hypro999 reassigned [PR #1](http://139.59.64.214:7990/projects\
+/SBOX/repos/sandbox/pull-requests/1) to no one"""
+        expected_topic = "sandbox / PR #1 Branch1"
+        self.send_and_test_stream_message("pull_request_remove_reviewer",
+                                          expected_topic,
+                                          expected_message)
```

Comment:
We should say something like "hypro999 removed all reviewers from PR #1".

Reply:
Done, (same reason as above for why something like this wasn't done earlier).